Kibana association E2E test is flaky

```
 --- FAIL: TestKibanaAssociationWithNonExistentES/Non_existent_backend_should_generate_event (300.00s)
        utils.go:80: 
            	Error Trace:	utils.go:80
            	Error:      	Received unexpected error:
            	            	event did not fire: AssociationError
            	Test:       	TestKibanaAssociationWithNonExistentES/Non_existent_backend_should_generate_event
```

--- a/test/e2e/kb/association_test.go
+++ b/test/e2e/kb/association_test.go
@@ -42,6 +42,7 @@ func TestCrossNSAssociation(t *testing.T) {
 func TestKibanaAssociationWithNonExistentES(t *testing.T) {
 	name := "test-kb-assoc-non-existent-es"
 	kbBuilder := kibana.NewBuilder(name).
+		WithElasticsearchRef(commonv1beta1.ObjectSelector{Name: "some-es"}).
 		WithNodeCount(1)
 
 	k := test.NewK8sClientOrFatal()

